modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/manager/v1_0/DefaultObjectEntryManagerImpl.java

```diff
@@ -400,22 +400,6 @@ public Page<ObjectEntry> getObjectEntries(
 			}
 		}
 
-		String[] selectedObjectFieldNames = null;
-
-		UriInfo uriInfo = dtoConverterContext.getUriInfo();
-
-		if (uriInfo != null) {
-			MultivaluedMap<String, String> queryParameters =
-				uriInfo.getQueryParameters();
-
-			String fields = queryParameters.getFirst("fields");
-
-			if (fields != null) {
-				selectedObjectFieldNames = StringUtil.split(
-					fields, StringPool.COMMA);
-			}
-		}
-
```

Comment:
I saw that you are deleting this part of the code, but I don't see it after the refactoring.
Could you please explain if this code was implemented in another method?

This code was added in this PR: https://github.com/brianchandotcom/liferay-portal/pull/149499
Can we make sure this behavior still works - https://liferay.atlassian.net/browse/LPD-22496 ?

A test was implemented in `ObjectEntryLocalServiceTest > testGetValuesList`. But I didn't see the results of this particular test method in the Jenkins Console in the [case result](https://testray.liferay.com/#/project/35392/routines/45357/build/71211731/case-result/71219140).

Can you double-check it, please?

Reply:
Hey Marcela! Thank you for your revision.

Before, only the selected fields where included in the query but at the end, when calling `_getObjectEntry`, only the primaryKey was needed:

		com.liferay.object.model.ObjectEntry serviceBuilderObjectEntry =
			_objectEntryService.getObjectEntry(
				GetterUtil.getLong(
					values.get(objectDefinition.getPKObjectFieldName())));

So now, we only include the primaryKey in the query, therefore as for the [LPD's ](https://liferay.atlassian.net/browse/LPD-22496) concern of performance issues, it is even more efficient.

Reply:
Thank you!